src/commands/saveDocumentScreenshot.js

```diff
@@ -1,11 +1,17 @@
 import makeDocumentScreenshot from '../modules/makeDocumentScreenshot';
 import saveBase64Image from '../utils/saveBase64Image';
 
-// Note: function name must be async to signalize WebdriverIO that this function returns a promise
-export default async function async(browser, fileName, options = {}) {
+/**
+ * @alias browser.saveDocumentScreenshot
+ * @param {string} fileName
+ * @param {Object} options
+ *
+ * Note: function name must be async to signalize WebdriverIO that this function returns a promise
```

Comment:
Can you move the `Note:...` into a normal comment (e.g. below function declaration)? Just to make sure that this will never show up in JSDoc cause this is more an internal comment than a public one.